pysrc/pip_resolve.py

```diff
@@ -202,12 +239,14 @@ def matches_environment(requirement):
             return match[0].lower() == sys_platform
     return True
 
+
 def is_testable(requirement):
-    return requirement.editable == False and requirement.vcs is None
+    return requirement.editable is False and requirement.vcs is None
+
 
 def detect_encoding_by_bom(path):
     with open(path, 'rb') as f:
-        raw = f.read(4)    # will read less if the file is smaller
+        raw = f.read(4)  # will read less if the file is smaller
```

Comment:
Magic number. Consider changing.

https://dev.to/producthackers/code-smell-magic-numbers-3ngc

Reply:
Good point, fixed in [7ba21cb](https://github.com/snyk/snyk-python-plugin/pull/234/commits/7ba21cb823349197437a93ed4395fa8ebd6ffaef) 🙏